libraries/AP_InertialSensor/AP_InertialSensor_SITL.cpp

```diff
@@ -170,17 +166,15 @@ void AP_InertialSensor_SITL::generate_accel()
             Vector3f centripetal_accel = angular_rate % (angular_rate % pos_offset);
 
             // apply corrections
-            xAccel += lever_arm_accel.x + centripetal_accel.x;
-            yAccel += lever_arm_accel.y + centripetal_accel.y;
-            zAccel += lever_arm_accel.z + centripetal_accel.z;
+            accel.x += lever_arm_accel.x + centripetal_accel.x;
+            accel.y += lever_arm_accel.y + centripetal_accel.y;
+            accel.z += lever_arm_accel.z + centripetal_accel.z;
```

Comment:
```suggestion
            accel += lever_arm_accel + centripetal_accel;
```

... 'though the fact you've done most of the other equivalents in this PR leads me to think this might not work for some reason...